hw_diag/tests/test_get_miner_diag.py

```diff
@@ -1,78 +1,45 @@
 import unittest
-
 from unittest.mock import patch
-from copy import deepcopy
+import grpc
+from concurrent import futures
 
-import hm_pyhelper.miner_json_rpc.client
+import hm_pyhelper
+from hm_pyhelper.protos import local_pb2_grpc
+from hm_pyhelper.tests.test_gateway_grpc import TestData, MockServicer
```

Comment:
Is it ok to reuse test data and code from hm_pyhelper? I can copy it but seemed like unncessary duplication.

Reply:
Seems reasonable to me.